lib/stylesheet/compiler.rb

```diff
@@ -65,14 +65,25 @@ def self.compile(stylesheet, filename, options = {})
       result = engine.render
 
       if options[:rtl]
-        require "r2"
-        [R2.r2(result), nil]
+        [rtlify_css(result) || result, nil]
       else
         source_map = engine.source_map
         source_map.force_encoding("UTF-8")
 
         [result, source_map]
       end
     end
+
+    def self.rtlify_css(src_css)
+      @context ||=
+        begin
+          context = MiniRacer::Context.new
```

Comment:
I think we should add a timeout here for safety. It can be quite high - just need a safety net for any pathalogical bugs. Would 10 seconds be reasonable?